### DISCUSSION TITLE
need help on opening Dbeaver on my mac OS yosemite 10.10.5

### DISCUSSION THREAD

**Discussion by kittknight** [dbeaver/dbeaver#22420] [Q&A]
I need help on how to open my Dbeaver installation.  IT shows error as follows:  The JVM shared library "/Applications/DBeaver.app/Contents/Eclipse/jre/Contents/Home/bin/../lib/server/libjvm.dylib"
does not contain the JNI_CreateJavaVM symbol.

Thank you in advance.

**Comment by LonwoLonwo** [comment]
Hello @kittknight 

Which DBeaver version do you try to install?

**Reply by LonwoLonwo** [reply]
Please check that you downloaded the correct mac installation file version.

**Reply by kittknight** [reply]
I tried to install the latest but then I tried to find an older version.  What version would be OK to run it and what java version should I need to install?  Also it says I need to edit the dbeaver.ini file and add some command line.  what to do ? pls help. thanks :)

**Reply by LonwoLonwo** [reply]
You can see two installation files for Mac on our website.

![2024-01-12 13_33_32-Download _ DBeaver Community — Mozilla Firefox](https://github.com/dbeaver/dbeaver/assets/45152336/f54374f5-503e-4703-b8e6-178ff6eba34e)

What processor do you have? Intell? Or Apple Silicon? Your installation version depends on it.

**Comment by kittknight** [comment]
Thanks very much for your help.  I downloaded the latest one shown on the
site for Community for MAC OS.  Please guide me on what to do step by
step.  Thank you once again.

Best regards,
Kitt

**Comment by kittknight** [comment]
Thank you.  Im using Intel chip
Here is the detail of my processor, etc:

*MAC OS Yosemite 10.10.5*

*MacBook Air (13-inch, Early 2015)*

*Processor:  2.2 GHz Intel Core i7*

*Memory:  8 GB 1600 MHz DDR3*

May I know what Java should I install?

Thanks again for your help.

best regards,
Albert
